source/CMakeLists.txt

```diff
@@ -124,9 +124,10 @@ list(APPEND OPCODE_CPP_DEPENDS ${GENERATOR_INC_FILE})
 # We need to wrap the .inc files with a custom target to avoid problems when
 # multiple targets depend on the same custom command.
 add_custom_target(core_tables
-	DEPENDS ${OPCODE_CPP_DEPENDS}
-	        ${CORE_TABLES_BODY_INC_FILE}
-		${CORE_TABLES_HEADER_INC_FILE})
+  DEPENDS ${OPCODE_CPP_DEPENDS}
+          ${CORE_TABLES_BODY_INC_FILE}
+          ${CORE_TABLES_HEADER_INC_FILE}
+          spirv-tools-tables)
```

Comment:
This seems okay, but can we remove `${CORE_TABLES_BODY_INC_FILE}` and `${CORE_TABLES_HEADER_INC_FILE}`?

They are part of `spirv-tools-tables`.